dhcpv6/ztp/parse_vendor_options.go

```diff
@@ -0,0 +1,74 @@
+package ztpv6
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/gorhamc/dhcp/dhcpv6"
+)
+
+var errVendorOptionMalformed = errors.New("malformed vendor option")
+
+type VendorData struct {
+	VendorName, Model, Serial string
+}
+
+// VendorDataV6 will try to parse dhcp6 Vendor Specific Information options data
+// looking for more specific vendor data (like model, serial number, etc).
+// If the options are missing we will just return an empty
+func VendorDataV6(packet dhcpv6.DHCPv6) ([]VendorData, error) {
```

Comment:
For consistency with the ztpv4 module lets call this `ParseVendorData`. The V6 is redundant since it is part of the dhcpv6 package already.

Reply:
Do we need separate functions for `VendorDataV6` and `parseV6VendorSpecificInfo`? Maybe we can combine them.

Reply:
Why does it returns `[]VendorData`? Shouldn't it return one VendorData per packet?